data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/BulkRetryStrategy.java

```diff
@@ -115,6 +117,22 @@ public final class BulkRetryStrategy {
     private final Counter bulkRequestNotFoundErrors;
     private final Counter bulkRequestTimeoutErrors;
     private final Counter bulkRequestServerErrors;
+    private static final Logger LOG = LoggerFactory.getLogger(BulkRetryStrategy.class);
+
+    class BulkOperationRequestResponse {
```

Comment:
This should be `static` since it doesn't need anything from the current `BulkRetryStrategy`.